### PR TITLE
dashboard frontend

### PR DESCRIPTION
**Pull request by velgardey**
completes the dashboard frontend ui and conflicts with dev are resolved and two shadcn components viz. card and button are added

Additional Packages : pnpm i @radix-ui/react-slot

### CODE REVIEW

File: src/app/dashboard/page.tsx [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+            {profile.name}
+          </h2>
+          <p className="mb-4 text-xl sm:text-2xl md:text-3xl lg:text-4xl text-cyan-400 sm:mb-6 break-words">
+            {profile.alias}
+          </p>
+        </div>
+      </div>
+      <div className={`mb-4 sm:mb-6 md:mb-8 lg:mb-10 space-y-3 sm:space-y-4 md:space-y-5 lg:space-y-6 ${outfit.className}`}>
+        <p className="flex items-center text-base sm:text-lg md:text-xl lg:text-2xl text-gray-300">
+          <svg
```
Comment (JDeepD) [inline src/app/dashboard/page.tsx:125]: Replace.

File: src/app/dashboard/page.tsx [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+
+  if (!dashboardData) {
+    return <div className="min-h-screen flex items-center justify-center bg-black text-white text-lg sm:text-xl md:text-2xl lg:text-3xl">LOADING</div>
+  }
+
+  return (
+    <div className={`min-h-screen bg-black text-white p-1 sm:p-2 md:p-4 lg:p-6 ${outfit.className}`}>
+      <main className="container mx-auto max-w-full xl:max-w-[1350px] 2xl:max-w-[1500px]">
+        <div className="mb-2 sm:mb-3 md:mb-4 lg:mb-6">
+          <Button variant="ghost" className="text-white bg-blue-900/70 hover:bg-blue-800/80 rounded-full px-1 sm:px-2 md:px-3 lg:px-4 py-1 sm:py-1.5 md:py-2 text-xs sm:text-sm md:text-base lg:text-lg flex items-center">
```
Comment (JDeepD) [inline src/app/dashboard/page.tsx:285]: FIx for mobile.

File: src/app/dashboard/page.tsx [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+    null,
+  );
+
+  useEffect(() => {
+    fetchDashboardData()
+      .then((data) => {
+        setDashboardData(data);
+      })
+      .catch((error) => {
+        console.error("Error fetching dashboard data:", error);
```
Comment (JDeepD) [inline src/app/dashboard/page.tsx:273]: `toast.error("")`

File: src/components/ui/button.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import * as React from "react"
```
Comment (JDeepD) [inline src/components/ui/button.tsx:1]: Remove shadcn.

File: src/components/ui/card.tsx [hunk 5]
```diff
@@ -0,0 +1,79 @@
+import * as React from "react"
```
Comment (JDeepD) [inline src/components/ui/card.tsx:1]: Remove

**Review by JDeepD — CHANGES_REQUESTED** [review]

**Comment by JDeepD** [comment]
@velgardey run `pnpm build` before pushing. Also, delete `pnpm-lockfile` and run `pnpm i` again. then push.

**Comment by velgardey** [comment]
Screenshot 2024-10-10 232037

@JDeepD The pages are properly being built locally

**Comment by JDeepD** [comment]
@velgardey Delete the `pnpm-lockfile` and do `pnpm i`. Push to see if build is fixed. Also, resolve the merge conflicts

**Pull request merged**